packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -444,7 +444,7 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
      * A reference to the underlying {@apilink RequestList} class that manages the crawler's {@apilink Request|requests}.
      * Only available if used by the crawler.
      */
-    requestList?: RequestList;
```

Comment:
Technically, this is breaking for anybody who extends `BasicCrawler` and accesses this. But anyone who does that should be able to deal with the change IMO.

Reply:
@B4nan is this cool?

Reply:
i guess it is fine, by breaking you mean only if they would depend on a specific API that is not part of the new interface, right?

Reply:
Exactly. It seems improbable to me that anyone would do that, but life is full of surprises.